src/java/picard/analysis/QualityScoreDistribution.java

```diff
@@ -48,11 +48,39 @@
  * @author Tim Fennell
  */
 @CommandLineProgramProperties(
-        usage = "Program to chart " +
-                "quality score distributions in a SAM or BAM file.",
-        usageShort = "Charts quality score distributions for a SAM or BAM file",
+        usage = "Program to chart quality score distributions in a SAM or BAM file.  Output options include plotting " +
+                "the distribution of all of the reads, only the aligned reads, reads that have passed the Illumina " +
+                "Chastity filter thresholds (1).  Chastity is defined as the ratio of the brightest base intensity " +
+                "divided by the sum of the brightest and second brightest base intensities. Clusters 'pass filter' if " +
+                "no more than 1 base call has a chastity value below 0.6 in the first 25 cycles (2).  This filtration " +
+                "process removes the least reliable clusters from the image analysis results. " +
+                "<br /> <br />" +
+                "If the original quality score (OQ) tag is present, histogram will plot both the OQ and the " +
```

Comment:
Maybe worth clarifying that this is for running on data that has been put through GATK base recalibration.